tests/testthat/test-input.R

```diff
@@ -648,9 +648,10 @@ test_that("destructure body", {
   json <- '{"a": 3, "b": 2}'
   res <- pr$request("POST", "/multiply", body = json)
   expect_equal(res$status, 200)
-  expect_equal(res$headers,
+  expect_equal(res$headers[c("Content-Type", "X-Porcelain-Validated")],
                list("Content-Type" = "application/json",
                     "X-Porcelain-Validated" = "true"))
```

Comment:
```suggestion
  expect_equal(res$headers[["Content-Type"]], "application/json")
  expect_equal(res$headers[["X-Porcelain-Validated"]], "true")
```

Maybe a bit subjective, but I find this more concise and less confusing.